openslides/agenda/static/js/agenda/projector.js

```diff
@@ -49,19 +50,48 @@ angular.module('OpenSlidesApp.agenda.projector', ['OpenSlidesApp.agenda'])
         $scope.updateCurrentListOfSpeakers = function () {
             $scope.agendaItem = CurrentListOfSpeakersItem.getItem($scope.currentListOfSpeakersReference);
         };
+
+        $scope.getTitle = function () {
```

Comment:
Ca we please use a different approach here? The motion specific code should not live here but somewhere in motion app. So we can maybe do something like this:
```javascript
$scope.getTitle = function () {
    if ($scope.agendaItem.content_object.getAgendaTitle) {
        return $scope.agendaItem.conten_object.getAgendaTitle();
    } else {
        return ...
    }
};
```
So every app and even plugins can define the title itself.